docs/integration.rst

```diff
@@ -237,230 +235,290 @@ Airflow can be configured to read and write task logs in Google Cloud Storage.
+     - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     - :doc:`How to use <howto/operator/gcp/cloud_storage_transfer_service>`
+     - :mod:`airflow.operators.s3_to_gcs`
+       :mod:`airflow.gcp.operators.cloud_storage_transfer_service`
+
+   * - `Google BigQuery <https://cloud.google.com/bigquery/>`__
+     - `Google BigQuery <https://cloud.google.com/bigquery/>`__
+     -
+     - :mod:`airflow.operators.bigquery_to_bigquery`
+
+   * - `Google BigQuery <https://cloud.google.com/bigquery/>`__
+     - `Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     -
+     - :mod:`airflow.operators.bigquery_to_gcs`
+
+   * - `BigQuery <https://cloud.google.com/bigquery/>`__
+     - `MySQL <https://www.mysql.com/>`__
+     -
+     - :mod:`airflow.operators.bigquery_to_mysql`
+
+   * - `Apache Cassandra <http://cassandra.apache.org/>`__
+     - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     -
+     - :mod:`airflow.operators.cassandra_to_gcs`
+
+   * - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     - `Google BigQuery <https://cloud.google.com/bigquery/>`__
+     -
+     - :mod:`airflow.operators.gcs_to_bq`
+
+   * - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     - :doc:`How to use <howto/operator/gcp/gcs_to_gcs>`,
+       :doc:`How to use <howto/operator/gcp/cloud_storage_transfer_service>`
+     - :mod:`airflow.operators.gcs_to_gcs`
+       :mod:`airflow.gcp.operators.cloud_storage_transfer_service`
+
+   * - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     - `Amazon Simple Storage Service (S3) <https://aws.amazon.com/s3/>`__
+     -
+     - :mod:`airflow.operators.gcs_to_s3`
+
+   * - Local
+     - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     -
+     - :mod:`airflow.operators.local_to_gcs`
+
+   * - `Microsoft SQL Server (MSSQL) <https://www.microsoft.com/pl-pl/sql-server/sql-server-downloads>`__
+     - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     -
+     - :mod:`airflow.operators.mssql_to_gcs`
+
+   * - `MySQL <https://www.mysql.com/>`__
+     - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     -
+     - :mod:`airflow.operators.mysql_to_gcs`
+
+   * - `PostgresSQL <https://www.postgresql.org/>`__
+     - `Google Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     -
+     - :mod:`airflow.operators.postgres_to_gcs`
+
+   * - SQL
+     - `Cloud Storage (GCS) <https://cloud.google.com/gcs/>`__
+     -
+     - :mod:`airflow.operators.sql_to_gcs`
+
+Google Discovery Operators
+""""""""""""""""""""""""""
+
+Airflow also has support for requesting data from any Google Service using
+`Discovery API <https://developers.google.com/discovery/>`__.
+
+The operator are defined in the :mod:`airflow.contrib.operators.google_api_to_s3_transfer` module.
```

Comment:
Can we have a section for `Transfer operators and hooks` here as well where we then list the `airflow.contrib.operators.google_api_to_s3_transfer` module? Just in case we get more than just "to s3"

Reply:
I did not put this operator in the table above because I think it requires a separate comment. Just writing "All services" would not be descriptive enough, but placing a longer description in the column spoils the look of the entire table. I think that this is a very valuable operator and it is worth highlighting it by creating a separate section.  Would you like me to put this operator in the table above?

Reply:
I totally agree. It does require a separate section. 

But I would place the discovery based operators inside a table, too.
For example:
```suggestion
.. list-table::
   :header-rows: 1

   * - Service name
     - Guide
     - Hook
     - Operators
     - Sensors

   * - `GoogleApiToS3Transfer`
     - :mod:`airflow.contrib.operators.google_api_to_s3_transfer`
```

Reply:
I have another idea. What do you think about it?
http://vast-chicken.surge.sh/integration.html#gcp-google-cloud-platform
![Screenshot 2019-09-15 at 17 06 51](https://user-images.githubusercontent.com/12058428/64923580-3c3dbf00-d7db-11e9-9daf-9ca0d405a5ce.png)

Reply:
I love it

Reply:
Other proposition: 
<img width="690" alt="Screenshot 2019-09-15 at 20 58 40" src="https://user-images.githubusercontent.com/12058428/64926248-b8e09580-d7fb-11e9-902d-5ea0425e11a1.png">
I slightly changed the description to make it easier to understand. It does not contain additional implementation details.

Reply:
I like that 👍

Could you also change the text a bit:
```
The operator are defined in the :mod:`airflow.contrib.operators.google_api_to_s3_transfer` module. They also use :class:`airflow.contrib.hooks.google_discovery_api_hook.GoogleDiscoveryApiHook` to communicate with
Google Services via the `Google API Python Client <https://github.com/googleapis/google-api-python-client>`__.
Please note that this library is in maintenance mode. Therefore it is recommended that you use the custom GCP Service
Operators for working with the Google Cloud Platform.
```
to
```
Those discovery-based operators use :class:`airflow.contrib.hooks.google_discovery_api_hook.GoogleDiscoveryApiHook` to communicate with
Google Services via the `Google API Python Client <https://github.com/googleapis/google-api-python-client>`__.`
Please note that this library is in maintenance mode hence it won't fully support GCP in the future. Therefore it is recommended that you use the custom GCP Service
Operators for working with the Google Cloud Platform.
```

WDYT?

Reply:
Yes, this works, too.

Reply:
I fully took into account your suggestions for changing the description.